agent/api/task/task.go

```diff
@@ -3006,9 +3006,9 @@ func (task *Task) RecordExecutionStoppedAt(container *apicontainer.Container) {
 		return
 	}
 	logger.Info("Essential container stopped; recording task stopped time", logger.Fields{
-		field.TaskID:    task.GetID(),
-		field.Container: container.Name,
-		field.Time:      now.String(),
+		field.TaskID:             task.GetID(),
+		field.Container:          container.Name,
+		field.ExecutionStoppedAt: now.String(),
```

Comment:
`field.ExecutionStoppedAt` does not seem to exist right now. Let's add this as a field and also delete `field.Time` if it is not used anywhere else (if it is use elsewhere, we might consider replacing those usage(s) with more unique field name(s) as well).